features/config/upgrade/src/main/resources/changelog-cm/29.0.0/changelog-cm.xml

```diff
@@ -30,4 +30,10 @@
         <cm:importConfig schemaId="jmxconfig" configId="default" filePath="jmx-config.xml"/>
     </changeSet>
 
+    <changeSet author="uguggilam" id="1.0-register-schema-notifd">
+        <cm:registerSchema id="notifd"/>
+        <cm:importSchemaFromXsd id="notifd" xsdFileName="notifd-configuration.xsd" xsdFileHash="ffb7114b6ac496314340fa5eb95b5bfdd8e7754c3c2916e618673802c913a5e2" rootElement="notifd-configuration"/>
+        <cm:importConfig schemaId="notifd" configId="default" filePath="notifd-configuration.xml"/>
+    </changeSet>
+
```

Comment:
I think you need to rebase, it is 30.0.0 in phase2 now.

Reply:
As discussed, 30.0.0 is in phase 1.